app/views/admin_request/edit.html.erb

```diff
@@ -64,6 +64,10 @@
     </span>
   </p>
 
+  <p><label for="info_request_prominence_reason"><strong>Reason for prominence</strong></label>
+    <%= text_area 'info_request', 'prominence_reason', rows: 5, class: 'span6' %>
+  </p>
+
```

Comment:
Let's add some help text here:

```diff
diff --git a/app/views/admin_request/edit.html.erb b/app/views/admin_request/edit.html.erb
index 7bbd773bb..056ed1b26 100644
--- a/app/views/admin_request/edit.html.erb
+++ b/app/views/admin_request/edit.html.erb
@@ -66,6 +66,11 @@
 
   <p><label for="info_request_prominence_reason"><strong>Reason for prominence</strong></label>
     <%= text_area 'info_request', 'prominence_reason', rows: 5, class: 'span6' %>
+
+    <span class="help-block">
+      This reason is shown in public. If left blank a generic reason of
+      "<%= default_prominence_reason %>" will be shown.
+    </span>
   </p>
 
   <p><label for="info_request_described_state"><strong>Described state</strong></label>
```

Will need the following change, which I think ought to be fine. Don't think it matters if admins generally see this text?

```diff
diff --git a/app/helpers/prominence_helper.rb b/app/helpers/prominence_helper.rb
index f92bbf13d..479f16de0 100644
--- a/app/helpers/prominence_helper.rb
+++ b/app/helpers/prominence_helper.rb
@@ -17,6 +17,11 @@ def render_prominence(object, format: :html)
       join(' ').squish.html_safe
   end
 
+  def default_prominence_reason
+    _("There are various reasons why we might have done this, sorry we can't " \
+      "be more specific here.")
+  end
+
   private
 
   def prominence_notice(object)
@@ -100,12 +105,6 @@ def prominence_reason_for_object(object)
     object.prominence_reason.presence || default_prominence_reason
   end
 
-  def default_prominence_reason
-    return '' if current_user&.is_admin?
-    _("There are various reasons why we might have done this, sorry we can't " \
-      "be more specific here.")
-  end
-
   def prominence_contact_us
     return if current_user&.is_admin?
     _('Please {{contact_us_link}} if you have any questions.',
```